pkgs/development/tools/kustomize/4.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "kustomize_4";
+  version = "4.5.7";
+  # rev is the commit of the tag, mainly for kustomize version command output
+  rev = "56d82a8378dfc8dc3b3b1085e5a6e67b82966bd7";
+
+  ldflags = let t = "sigs.k8s.io/kustomize/api/provenance"; in
+    [
+      "-s"
+      "-X ${t}.version=${version}"
+      "-X ${t}.gitCommit=${rev}"
+    ];
+
+  src = fetchFromGitHub {
```

Comment:
Usually this goes right after `version`.

Refs: https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/attribute-ordering.md

Reply:
I'm happy to rearrange it; I just started with the way `3.nix` and `default.nix` do things.

Should I reorder all three files?

Reply:
@AndersonTorres I think I've fixed the ordering in my addition.

Reply:
Let's ignore this for now.